src/pymovements/utils/archives.py

```diff
@@ -42,6 +44,7 @@ def extract_archive(
         remove_finished: bool = False,
         remove_top_level: bool = True,
         verbose: int = 1,
+        resume: bool = False,
```

Comment:
verbose should come after resume

Reply:
which means this change would be a breaking change as `verbose` can be passed as a positional argument.

I strongly prefer `verbose` as the last argument. We should probably but a `*, ` after the recursive parameter and enforce passing `remove_finished`, `remove_top_level`, `resume` and `verbose` as keyword arguments.

We can either introduce the breaking in this PR or in a separate one. What do you think?

Also we should keep this issue in mind when enhancing signatures in the future. To be future-proof and backwards compatible it is necessary to keep the number of positional arguments to a comfortable minimum.